examples/utils/webpack-fluid-loader/package.json

```diff
@@ -149,7 +149,7 @@
 		}
 	},
 	"typeValidation": {
-		"broken": {},
-		"disabled": true
+		"disabled": true,
```

Comment:
Probably worth calling out in the PR description. Disabled since this is now a @fluid-examples package, right?

Reply:
Type tests were disabled for this package prior to this change.